File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/ExternalProcessStep.ts
import { ProcessStepType } from '../enums';
import { ProcessStep } from './ProcessStep';
import { GuidValue } from '@omnia/fx-models';

export interface ExternalProcessStep extends ProcessStep {
    type: ProcessStepType.External;

    rootProcessStepId: GuidValue;

    //client-side.
    //Use for shape that link to external-process
    basedProcessStepId?: GuidValue;
}

File: src/Omnia.ProcessManagement.Models/Processes/ExternalProcessStep.cs
using Omnia.Fx.Models.Language;
using Omnia.ProcessManagement.Models.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class ExternalProcessStep : ProcessStep
    {
        public override ProcessStepType Type => ProcessStepType.External;
        public Guid OPMProcessId { get; set; }
    }
}